lib/static.js

```diff
@@ -50,6 +50,36 @@ exports.load = function(content, options) {
 };
 
 /**
+* $.stream([cb], [options])
+*/
+
+exports.stream = function(cb, options) {
+  if(typeof cb !== 'function'){
+    options = cb;
+    cb = null;
+  }
+
+  var stream = parse.createWriteStream(function done(err, dom) {
+    if (err) {
+      stream.emit('error', err);
+    } else {
+      var loaded = exports.load(dom, options);
+      stream.emit('finish', loaded);
+
+      if(typeof cb === 'function'){
+        cb(null, loaded);
+      }
+    }
+  }, options);
+
+  if(typeof cb === 'function'){
+    stream.on('error', cb);
```

Comment:
The first argument of the callback is supposed to be an error object, the callback would need to be wrapped anyway. Adding it to the stream callback seems to be the best idea.